tools/commit-checker/check.py

```diff
@@ -461,7 +461,7 @@ def run_section(section, out) -> int:
         global commits
         count = 0
         if section == CLOSED_NO_COMMIT:
-            print("Tip: Tickets with type 'Task' or 'User Guide' or resolution 'Fixed by Other Ticket' are ignored.", file=out)
+            print("Tip: Change the ticket type or set the resolution to one such as 'Fixed by other ticket' or 'Fix Non-repo' if commits aren't expected.", file=out)
```

Comment:
```suggestion
            print("Tip: If commits aren't expected, change the ticket type to 'Task' or 'User Guide' or set the resolution to one such as 'Fixed by other ticket' or 'Fix Non-repo'.", file=out)
```

Reply:
We might finally need a CLDR mode.

Reply:
CLDR Tasks require commits (they are more like a "chore") unless it's closed as non repo.

Reply:
Suggestion: Print two lines of tips, one with a "CLDR tip:" and one with an "ICU tip:"

Reply:
@sffc @markusicu  PTAL

Reply:
Less specific than what @sffc had requested (suggesting specific ticket types). Shane: ok?

Reply:
@markusicu sorry, user error here on the git side.  Should be exactly Shane's text.

Reply:
@markusicu @sffc  PTAL.  The wording should match Shane's request under the ICU tip.